components/panel/_panel.scss

```diff
@@ -20,6 +20,10 @@
       color: white;
     }
   }
+  /* Removes inherited bottom margin to make whitespace inside panel equal */
+   *:last-child {
```

Comment:
This will match _any_ last-child element, including inside other containers in the `.panel`. I think `>*:last-child` would be better as it just matches the last-child direct descendant.

Reply:
Typical - I even had `>*` at one point before commit but didn't match a content test to it and removed again. I'll add it back in there.